### PR TITLE
Wishlist Screen

### PR DESCRIPTION
**Pull request by MarkVed17**
- Added Wishlist Card
- Added functionality to `add/ remove from wishlist` on Product Card
- Added `remove from wishlist` functionality on Wishlist Card

Preview:
recording

### CODE REVIEW

File: src/frontend/services/wishlist-services/getWishlistService.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export const getWishlistService = async (token) => {
+  try {
+    const { data } = await axios.get("/api/user/wishlist", {
+      headers: {
+        authorization: token,
+      },
+    });
+    return data.wishlist;
+  } catch (error) {
+    console.error(error.message);
```
Comment (AbhishekkGautam) [inline src/frontend/services/wishlist-services/getWishlistService.js:12]: We should remove console based methods before commit. We can store error in state & show it on view.

**Review by AbhishekkGautam — APPROVED** [review]
Code looks clean & well structured. 
Price can be formatted as `3,000`. You can use `Intl.NumberFormat()` (mdn) to put commas in price.

File: src/frontend/components/ProductsListing/ProductsListing.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { ProductCard } from "../ProductCard/ProductCard";
 import "./ProductsListing.css";
 
-function ProductsListing({finalProducts}) {
+function ProductsListing({ finalProducts }) {
   // const { products } = useProducts();
```
Comment (horsemaker) [inline src/frontend/components/ProductsListing/ProductsListing.js:6]: Comment

**Review by horsemaker — APPROVED** [review]
Looks good! 👌

**Review by himadri2110 — APPROVED** [review]
Everything looks good. Well structured code.

**Comment by MarkVed17** [comment]
> Code looks clean & well structured. Price can be formatted as `3,000`. You can use `Intl.NumberFormat()` (mdn) to put commas in price.

I'll try this. Thanks!

**Pull request merged**